openapi.yaml

```diff
@@ -839,23 +839,11 @@ paths:
             format: uuid
```

Comment:
I think the correct definition for a required empty filter object of `{ }` with optional parameters would be this
```
      requestBody:
        required: true
        content:
          application/json:
            schema:
              type: object
              minProperties: 0
              properties:
                business_id:
                  type: string
                  format: uuid
                survey:
                  type: string
                  format: uuid
                status:
                  type: string
                  enum: ["ENABLED", "DISABLED"]
```

Reply:
get requests can't have requestBody according to my api editor

Reply:
> get requests can't have requestBody according to my api editor

As discussed, that's probably best practice and technically correct (albeit not strictly enforced by the frameworks). As there are only three optional parameters they could be moved to the query string. Happy to raise a separate card for this.